### ISSUE TITLE
Custom headers

### ISSUE THREAD

**Issue by larsderidder** [AxisCommunications/media-stream-library-js#720]
Is it possible to send a custom header along? Cameras are hosted behind a reverse proxy which requires an authorization header.

**Comment by steabert** [comment]
> Is this library still being maintained?

Absolutely, although resources for replying to issues and feature requests can be variable leading to somethings very slow response. We rely on the community to help each other out or make changes to the code.
